cfme/containers/image.py

```diff
@@ -59,6 +59,43 @@ def perform_smartstate_analysis(self, wait_for_finish=False):
                 raise TimedOutError('Timeout exceeded, Waited too much time for SSA to finish ({}).'
                                     .format(ssa_timeout))
 
+    def check_compliance(self, timeout=240):
+        """Initiates compliance check and waits for it to finish."""
+        original_state = self.compliance_status
+        tb.select('Policy',
```

Comment:
**Required**
There's no navigation here, so you're relying on the self.compliance_status call to navigate for you. I don't think this navigation dependency is a good thing to have, and you should navigate explicitly here to the page where you want to click the toolbar.